src/nav/telemetry-data-platform.yml

```diff
@@ -137,64 +167,34 @@ pages:
                 path: /docs/apis/nerdgraph/examples/export-import-dashboards-using-api
               - title: Managing live chart URLs
                 path: /docs/apis/nerdgraph/examples/manage-live-chart-urls-via-api
-          - title: 'NerdGraph: alerting'
+          - title: 'Distributed tracing'
             pages:
-              - title: 'Applied intelligence: destinations'
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-destinations
-              - title: 'Applied intelligence: channels'
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels
-              - title: 'Applied intelligence: workflows'
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-workflows
-              - title: NRQL alert conditions
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts
-              - title: Alerts policies
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-alerts-policies
-              - title: Alert notification channels
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-alert-notification-channels
-              - title: Alerting loss of signal
-                path: /docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling
-              - title: Nerdgraph tutorial
-                path: /docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github
-              - title: Incident correlation topology
-                path: /docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial
-          - title: 'NerdGraph: logs'
+              - title: View trace details
```

Comment:
Nav title: `View trace details`
Doc title: `NerdGraph tutorial: Viewing distributed trace details`
For consistency, I'd write or `view` or `viewing`.

Reply:
Nice work!